### ISSUE TITLE
Cross-browser test results of tests in a PR are not very discoverable

### ISSUE THREAD

**Issue by zcorpan** [web-platform-tests/wpt.fyi#1307]
So I wanted to find out what the test results were for https://github.com/web-platform-tests/wpt/pull/14824 and clicked around in the links from the PR's status checks.

https://github.com/web-platform-tests/wpt/pull/14824/checks?check_run_id=50686531

The last link there ("Latest results for c70defa") is what I wanted, but I didn't get to that one before I gave up and asked @jugglinmike if he knew if what I was looking for existed. He found it by searching, so maybe I just suck at finding things. But then again, if I can't find this when I know the test results are there, how do we expect other contributors to find it?

Can we make it more discoverable?

cc @Hexcles

**Comment by foolip** [comment]
@lukebjerring this is about the wpt.fyi checks. There may be other open issues about this, but I've experienced the same thing when reviewing IDL update PRs, that it takes quite a bit of time to go through the results and comparing cross-browser results is tricky.

A single check would probably address this, but there might be other ways too.

**Comment by lukebjerring** [comment]
> Can we make it more discoverable?

Do we have any suggestions here?
Maybe a different name for the link? Another, separate, status check, for interoperability?

**Comment by zcorpan** [comment]
> Another, separate, status check, for interoperability?

Yes, and putting it first (or at least early) in the list, I think would make it a lot more discoverable.

**Comment by foolip** [comment]
The order of checks unfortunately I think can't be controlled, but one could try to reduce the total number of checks to make it easier.
